src/hashTable.js

```diff
@@ -0,0 +1,64 @@
+export default class HashTable {
+  constructor() {
+    this.hashTable = []
+    this.count = 0
+  }
+
+  hash( key ) {
+    let sum = 0;
+    const randomH = 31
+    const prime = 51
+    key = key.toString();
+    for (let i = 0; i < key.length; i++) {
+        sum = (randomH * sum) + key.charCodeAt(i)
+    }
+    return sum % prime // return hashingIndex
+  }
+
+  put( key, value ) {
+    const hashedKey = this.hash( key )
+    this.hashTable[hashedKey] = { 'key': key, 'value':value }
```

Comment:
You could use implicit object creation to make this more concise:
```js
this.hashTable[ hashedKey ] = { key, value }
```
Since the `hashedKey` constant is only used once, I would consider inlining it as well:
```js
this.hashTable[ this.hash( key ) ] = { key, value }
```